src/components/AccountForm.vue

```diff
@@ -228,13 +228,13 @@
 		</tabs>
 		<slot name="feedback"></slot>
 		<input type="submit" class="primary" :disabled="loading" :value="submitButtonText" @click="onSubmit" />
```

Comment:
Be careful about the double submit now, because an input[type=submit] will trigger a click and then the form submit :)

Reply:
daaaaaaaaaayum